monai/transforms/croppad/array.py

```diff
@@ -58,28 +58,33 @@ class SpatialPad(Transform):
     for additional details.
 
     Args:
-        c: the spatial size of output data after padding, if a dimension of the input
+        spatial_size: the spatial size of output data after padding, if a dimension of the input
             data size is bigger than the pad size, will not pad that dimension.
-            If its components have non-positive values, the corresponding size of input image will be used (no padding).
-            for example: if the spatial size of input data is [30, 30, 30] and `spatial_size=[32, 25, -1]`,
-            the spatial size of output data will be [32, 30, 30].
+            If its components have non-positive values, the corresponding size of input image will be used
+            (no padding). for example: if the spatial size of input data is [30, 30, 30] and
+            `spatial_size=[32, 25, -1]`, the spatial size of output data will be [32, 30, 30].
         method: {``"symmetric"``, ``"end"``}
             Pad image symmetric on every side or only pad at the end sides. Defaults to ``"symmetric"``.
         mode: {``"constant"``, ``"edge"``, ``"linear_ramp"``, ``"maximum"``, ``"mean"``,
             ``"median"``, ``"minimum"``, ``"reflect"``, ``"symmetric"``, ``"wrap"``, ``"empty"``}
             One of the listed string values or a user supplied function. Defaults to ``"constant"``.
             See also: https://numpy.org/doc/1.18/reference/generated/numpy.pad.html
+        kwargs: other args for `np.pad` API, note that `np.pad` treats channel dimension as the first dimension.
```

Comment:
could be `pad_kwargs` or `np_kwargs`

Reply:
OK, let me change it to `np_kwargs` as it's used for numpy API and this class is `SpatialPad`, no need to use the word `pad`.

Thanks.

Reply:
updated in the latest commit.
Thanks.